student.rb

```diff
@@ -1,6 +1,8 @@
-require 'person'
+require_relative 'person'
 
 class Student < Person
```

Comment:
When listing all the people, it shows `name` as `unknown` for students. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/33268927/218275003-77cafddd-c278-42f7-a620-8de6739df2ae.png)